packages/storybook/stories/INTERNAL/CallReadiness/DevicePermissionDropdown/DevicePermissionDropdown.stories.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Icon, IDropdownStyles, mergeStyles } from '@fluentui/react';
+import { useTheme, _DevicePermissionDropdown } from '@internal/react-components';
+import { Meta } from '@storybook/react/types-6-0';
+import React from 'react';
+import {
+  _DevicePermissionDropdownProps,
+  _DevicePermissionDropdownStrings
+} from '../../../../../react-components/src/components/DevicePermissionDropdown';
+import { COMPONENT_FOLDER_PREFIX } from '../../../constants';
+import { hiddenControl } from '../../../controlsUtils';
+
+const DevicePermissionDropdownStory = (): JSX.Element => {
+  const theme = useTheme();
+
+  const devicePermissionDropdownStrings: _DevicePermissionDropdownStrings = {
+    label: 'Camera',
+    placeHolderText: 'Enable Camera (optional)',
```

Comment:
Out of interest, what happens to text that's very long?

Reply:
It get cut off , but we can make this better by playing around with styling 
<img width="358" alt="Screen Shot 2022-09-28 at 11 28 31 AM" src="https://user-images.githubusercontent.com/96077406/192860621-be5f5a6c-4110-447c-b61c-86138343f5ad.png">